Revit_Core_Engine/Modify/SortColumEndpoints.cs

```diff
@@ -25,34 +25,33 @@
 using BH.oM.Geometry;
 using System.ComponentModel;
 
-namespace BH.Engine.Adapters.Revit
+namespace BH.Revit.Engine.Core
 {
     public static partial class Modify
     {
         /***************************************************/
-        /****              Public methods               ****/
+        /****               Public Methods              ****/
         /***************************************************/
 
-        [Description("Flip the input line if its start point is higher than its endpoint.")]
-        [Input("line", "Line to check.")]
-        [Output("line", "The original line if its start point is already below its endpoint. Otherwise, the line flipped.")]
-        public static Line SortEndpoint(this Line locationLine)
+        [Description("Flip the input line of a BHoM Column object if its start point is higher than its endpoint. This allows pushing it to Revit")]
+        [Input("line", "Column line to check.")]
+        [Output("line", "The original column line if its start point is already below its endpoint. Otherwise, the same line flipped.")]
+        public static Line SortColumEndpoints(this Line columnLine)
```

Comment:
Both file and method names have a typo 🙈 

Besides that welcome to the swamp of a large code base! With your current implementation, there would be 2 errors in case of both points on the same elevation:
- one on the level of this method, with no info about the element it refers to
- one on the convert level, with the info about the element it refers to

Not ideal. Since you took this slightly more tricky route, I would go a step further now and rename the method to something like `BH.Revit.Engine.Core.Query.ColumnCurve`, taking the `IFramingElement` and returning the line. Inside you would do all necessary checks: if the framing element has location, if the location is a line, if it is flipped etc., everything that is currently done in the convert method. Then you could also manage the errors/warnings/notes better, plus unify the inputs for both . What do you think?